pkg/pool-gyro/contracts/Gyro2CLPPoolFactory.sol

```diff
@@ -60,20 +74,31 @@ contract Gyro2CLPPoolFactory is BasePoolFactory {
 
         pool = _create(
             abi.encode(
-                IGyro2CLPPool.GyroParams({ name: name, symbol: symbol, sqrtAlpha: sqrtAlpha, sqrtBeta: sqrtBeta }),
+                IGyro2CLPPool.GyroParams({
+                    name: name,
+                    symbol: symbol,
+                    sqrtAlpha: sqrtAlpha,
+                    sqrtBeta: sqrtBeta,
+                    version: _poolVersion
+                }),
                 getVault()
             ),
             salt
         );
 
+        LiquidityManagement memory liquidityManagement = getDefaultLiquidityManagement();
```

Comment:
It's really straightforward, but maybe add tests that set the LM flags on creation and check that they're set?

Reply:
lol this actually was incorrect in the eclp

Reply:
https://github.com/balancer/balancer-v3-monorepo/pull/1233/commits/87c33129033f533a3eb6a03d75ca283fccdd22d8#diff-9922de2898f2299995ee3dc26bf4a0ef83c04ca3523ed4d556310e5642d9e027R99